### ISSUE TITLE
Semi-Annual Channel - Bug Fix delays

### ISSUE THREAD

**Issue by dhoegl** [MicrosoftDocs/OfficeDocs-DeployOffice#128]
According to this document, the timing for bug fixes is missing. PLEASE add a qualifying NOTE under Semi-Annual Channel stating similar to "IMPORTANT: Bug Fixes are released with Feature Updates. For Semi-Annual Channel customers, it is possible that a Fix will not be deployed for up to six months (Jan or June each year), until the version with the Fix is available from Microsoft. In other words, if a Bug is found in the version deployed, Semi-Annual Channel customers may continue to experience the Bug until the next Feature Update (up to 6 months later for Semi-Annual Channel)."

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0cd85e3b-e780-457e-9502-856bc6315bca
* Version Independent ID: 416e351f-13c5-e951-ea81-1650910a7c25
* Content: [Overview of update channels for Office 365 ProPlus](https://docs.microsoft.com/en-us/deployoffice/overview-of-update-channels-for-office-365-proplus)
* Content Source: [DeployOffice/overview-of-update-channels-for-office-365-proplus.md](https://github.com/MicrosoftDocs/OfficeDocs-DeployOffice/blob/live/DeployOffice/overview-of-update-channels-for-office-365-proplus.md)
* Service: **o365-proplus-itpro**
* GitHub Login: @DHB-MSFT
* Microsoft Alias: **danbrown**

**Comment by DHB-MSFT** [comment]
That's not actually accurate. Semi-Annual Channel can get bug fixes at times other than January and July. For example, 2 Office suite bug fixes were included in the Semi-Annual Channel release earlier this week -- [https://docs.microsoft.com/en-us/officeupdates/semi-annual-channel-2018#version-1803-october-9](https://docs.microsoft.com/en-us/officeupdates/semi-annual-channel-2018#version-1803-october-9). But there is a very high bar for what fixes go into Semi-Annual Channel, Only very critical/serious bug fixes are considered for Semi-Annual Channel, based on enterpriese customer feedback and product team input.  Hope that explanation helps.

**Comment by dhoegl** [comment]
Hi DHB-MSFT... This is interesting information - actually very positive news. However, I've had several calls with our O365 Escalation Engineers over the past few days, and they continued to reiterate that bug fixes are never pushed out-of-band, and fall into the monthly Feature Updates. Can you point me to any documentation stating differently? Thanks!

**Comment by DHB-MSFT** [comment]
The last sentence of the [Semi-Annual Channel](https://docs.microsoft.com/en-us/deployoffice/overview-of-update-channels-for-office-365-proplus#semi-annual-channel-for-office-365-proplus) section says " If there is a critical non-security issue that needs to be fixed in Semi-Annual Channel, that non-security update will also be included in a Semi-Annual Channel release."   Bug fixes in Semi-Annual Channel do tend to be rare, because the idea is that any critical bugs are caught and fixed in Monthly Channel or Semi-Annual Channel (Targeted) first before that version of the code every gets released in Semi-Annual Channel.  Hope that helps.

**Comment by dhoegl** [comment]
Thank you for the IM chat - I believe the changes to the document will greatly help our customers understand how these issues (critical and non-critical, non-security) are handled.
Thanks again!

**Comment by DHB-MSFT** [comment]
@dhoegl -- thanks for the feedback. I've made some updates to the article, which I hope will clarify things. See the end of the [Semi-Annual Channel](https://docs.microsoft.com/en-us/DeployOffice/overview-of-update-channels-for-office-365-proplus#semi-annual-channel-for-office-365-proplus) section and the [Frequency of updates](https://docs.microsoft.com/en-us/DeployOffice/overview-of-update-channels-for-office-365-proplus#frequency-of-updates-for-the-update-channels-for-office-365-proplus) section. Part of the confusion might have been with the table in the "Frequency of updates" section which had "Every six months" for the "Non-security updates" column for Semi-Annual Channel. I've changed that to "Monthly" and added more context in the sentences after the table. Hope that helps.
